mscore/pianoroll.cpp

```diff
@@ -512,13 +514,6 @@ void PianorollEditor::moveLocator(int i, const Pos& pos)
 void PianorollEditor::cmd(QAction* a)
       {
       score()->startCmd();
```

Comment:
Well there's no use calling startCmd, but it should be nice to have it since it is "cmd" function.

well at least I have to "setStaff" :)

I might want to keep this function alive since there might be shortcuts that should be defined there